.pre-commit-config.yaml

```diff
@@ -1,28 +1,26 @@
 exclude: |
   (?x)^(
-    tests/utils.py |
-    .devcontainer/
+    tests/utils.py
```

Comment:
This is the only file excluded because otherwise it will fail, since it doesn't follow the correct filename for test file, which should be like `test*.py`. Maybe testing utilities can move to the main code `tests` module?

Reply:
These utils are solely relevant for testing so to me it makes sense to keep them with the tests and just have a config line excluding them. What do you think @AlexandreAdam ?

Reply:
Great! It that's the case, then sounds good to me 😄